<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserGUI</class>
 <widget class="QWidget" name="UserGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>433</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>544</width>
    <height>433</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>544</width>
    <height>433</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>User Mode</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Resources/user.ico</normaloff>Resources/user.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(255,228,225)</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>520</width>
     <height>401</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="searchLineEdit">
       <property name="styleSheet">
        <string notr="true">background: rgb(255,255,255)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="styleSheet">
        <string notr="true">background: rgb(248,248,255)</string>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="movieLabel">
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="styleSheet">
        <string notr="true">background: rgb(248,248,255)</string>
       </property>
       <property name="text">
        <string>Add movie</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="styleSheet">
        <string notr="true">background: rgb(248,248,255)</string>
       </property>
       <property name="text">
        <string>Next movie</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QListWidget" name="showListWidget">
       <property name="styleSheet">
        <string notr="true">background: rgb(255,255,255)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="styleSheet">
        <string notr="true">background: rgb(255,255,255)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="showButton">
       <property name="styleSheet">
        <string notr="true">background: rgb(248,248,255)</string>
       </property>
       <property name="text">
        <string>Show</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="styleSheet">
        <string notr="true">background: rgb(248,248,255)</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openButton">
       <property name="styleSheet">
        <string notr="true">background: rgb(248,248,255)</string>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="styleSheet">
        <string notr="true">background: rgb(248,248,255)</string>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="styleSheet">
        <string notr="true">background: rgb(248,248,255)</string>
       </property>
       <property name="text">
        <string>EXIT</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
